--- xuidesigner-9999/Makefile.orig	2024-10-08 02:09:02.564257339 +0200
+++ xuidesigner-9999/Makefile	2024-10-08 02:10:34.082330630 +0200
@@ -5,6 +5,8 @@
 RED =  "\033[1;31m"
 NONE = "\033[0m"
 
+MYCFLAGS = $(CFLAGS)
+
 SUBDIR := XUiDesigner
 
 .PHONY: $(SUBDIR) libxputty  recurse
@@ -23,7 +25,7 @@
 
 libxputty: check-and-reinit-submodules
 ifneq ($(MAKECMDGOALS),debug)
-	@exec $(MAKE) --no-print-directory -j 1 -C $@ $(MAKECMDGOALS) CFLAGS='-O3 -D_FORTIFY_SOURCE=2 -Wall \
+	@exec $(MAKE) --no-print-directory -j 1 -C $@ $(MAKECMDGOALS) CFLAGS='$(MYCFLAGS) -D_FORTIFY_SOURCE=2 -Wall \
 	-fstack-protector -fno-ident -fno-asynchronous-unwind-tables -s -DNDEBUG -Wno-unused-result'
 else
 	@exec $(MAKE) --no-print-directory -j 1 -C $@ $(MAKECMDGOALS)
