--- source/Makefile.orig	2019-08-26 22:31:45.806757946 +0200
+++ source/Makefile	2019-08-26 22:33:09.711754885 +0200
@@ -33,7 +33,7 @@
 
 ZITA-AT1_O = zita-at1.o styles.o jclient.o mainwin.o png2img.o guiclass.o \
              button.o rotary.o tmeter.o retuner.o
-zita-at1:	CPPFLAGS += $(shell pkgconf --cflags freetype2)
+zita-at1:	CPPFLAGS += $(shell pkg-config --cflags freetype2)
 zita-at1:	LDLIBS += -lclxclient -lclthreads -lzita-resampler -lcairo \
 	-lfftw3f -ljack -lpthread -lpng -lXft -lX11 -lrt
 zita-at1:	$(ZITA-AT1_O) 
