diff -ru tuxguitar-1.6.0-b/TuxGuitar-synth-gervill/share/tuxguitar-synth-gervill.cfg tuxguitar-1.6.0/TuxGuitar-synth-gervill/share/tuxguitar-synth-gervill.cfg
--- tuxguitar-1.6.0-b/TuxGuitar-synth-gervill/share/tuxguitar-synth-gervill.cfg	2023-09-28 17:35:24.852313001 +0200
+++ tuxguitar-1.6.0/TuxGuitar-synth-gervill/share/tuxguitar-synth-gervill.cfg	2023-09-28 17:35:38.009312020 +0200
@@ -1,2 +1,2 @@
 ## Default Settings ##
-tuxguitar-synth-gervill.soundbank.path=${tuxguitar.home.path}/share/soundfont/MagicSFver2.sf2
+tuxguitar-synth-gervill.soundbank.path=/usr/share/sounds/sf2/FluidR3_GM.sf2
diff -ru tuxguitar-1.6.0-b/build-scripts/common-resources/common-linux/dist/tuxguitar-fluidsynth.cfg tuxguitar-1.6.0/build-scripts/common-resources/common-linux/dist/tuxguitar-fluidsynth.cfg
--- tuxguitar-1.6.0-b/build-scripts/common-resources/common-linux/dist/tuxguitar-fluidsynth.cfg	2023-09-28 17:35:24.915312996 +0200
+++ tuxguitar-1.6.0/build-scripts/common-resources/common-linux/dist/tuxguitar-fluidsynth.cfg	2023-09-28 17:35:38.008312020 +0200
@@ -1,7 +1,7 @@
 #tuxguitar-fluidsynth
 #Fri Oct 30 14:24:16 ART 2015
 soundfont.count=1
-soundfont.path0=${tuxguitar.home.path}/share/soundfont/MagicSFver2.sf2
+soundfont.path0=/usr/share/sounds/sf2/FluidR3_GM.sf2
 
 synth.gain=1.1
 synth.chorus.active=yes
diff -ru tuxguitar-1.6.0-b/build-scripts/common-resources/common-linux/dist/tuxguitar-jsa.cfg tuxguitar-1.6.0/build-scripts/common-resources/common-linux/dist/tuxguitar-jsa.cfg
--- tuxguitar-1.6.0-b/build-scripts/common-resources/common-linux/dist/tuxguitar-jsa.cfg	2023-09-28 17:35:24.915312996 +0200
+++ tuxguitar-1.6.0/build-scripts/common-resources/common-linux/dist/tuxguitar-jsa.cfg	2023-09-28 17:35:38.009312020 +0200
@@ -1,3 +1,3 @@
 #tuxguitar-jsa
 #Mon Nov 02 20:00:23 ART 2015
-soundbank.custom.path=${tuxguitar.home.path}/share/soundfont/MagicSFver2.sf2
+soundbank.custom.path=/usr/share/sounds/sf2/FluidR3_GM.sf2
diff -ru tuxguitar-1.6.0-b/build-scripts/tuxguitar-linux-swt-x86_64/pom.xml tuxguitar-1.6.0/build-scripts/tuxguitar-linux-swt-x86_64/pom.xml
--- tuxguitar-1.6.0-b/build-scripts/tuxguitar-linux-swt-x86_64/pom.xml	2023-09-28 17:35:24.926312996 +0200
+++ tuxguitar-1.6.0/build-scripts/tuxguitar-linux-swt-x86_64/pom.xml	2023-09-28 17:36:04.717310028 +0200
@@ -343,9 +343,6 @@
 								<copy todir="${project.build.directory}/${project.finalName}/dist">
 									<fileset dir="${project.rootPath}/TuxGuitar/dist/" />
 								</copy>
-								<copy todir="${project.build.directory}/${project.finalName}/share/soundfont" overwrite="true">
-									<fileset dir="${project.rootPath}/TuxGuitar-resources/resources/soundfont" />
-								</copy>
 
 								<copy todir="${project.build.directory}/${project.finalName}" overwrite="true">
 									<fileset dir="../common-resources/common" />
