diff -ru rsound-9999.old/patches/alsa/Makefile rsound-9999/patches/alsa/Makefile
--- rsound-9999.old/patches/alsa/Makefile	2023-10-05 17:58:39.162759290 +0200
+++ rsound-9999/patches/alsa/Makefile	2023-10-05 18:05:47.920727317 +0200
@@ -10,14 +10,14 @@
 all: $(TARGET)
 
 %.o: %.c
-	$(CC) -o $@ -c $< $(INCDIRS) $(CFLAGS) -std=gnu99 -Wall -O2 -g -pthread -fPIC -DPIC
+	$(CC) -o $@ -c $< $(INCDIRS) $(CFLAGS) -std=gnu99 -Wall -pthread -fPIC -DPIC
 
 $(TARGET): $(OBJ)
 	$(CC) -shared -o $@ $< -Wl,-no-undefined -pthread $(LIBS) -fPIC
 
 install: $(TARGET)
-	mkdir -p $(DESTDIR)$(PREFIX)/lib/alsa-lib
-	install -m755 $(TARGET) $(DESTDIR)$(PREFIX)/lib/alsa-lib
+	mkdir -p $(DESTDIR)$(PREFIX)/lib64/alsa-lib
+	install -m755 $(TARGET) $(DESTDIR)$(PREFIX)/lib64/alsa-lib
 
 uninstall:
 	rm -f $(DESTDIR)$(PREFIX)/alsa-lib/$(TARGET)
diff -ru rsound-9999.old/patches/vlc/Makefile rsound-9999/patches/vlc/Makefile
--- rsound-9999.old/patches/vlc/Makefile	2023-10-05 17:58:39.162759290 +0200
+++ rsound-9999/patches/vlc/Makefile	2023-10-05 18:06:38.088723575 +0200
@@ -20,15 +20,15 @@
 	$(CC) -o $@ -shared -fPIC $< -lrsound $(VLC_LIBS)
 
 %.o: %.c
-	$(CC) -c -o $@ $< $(CFLAGS) -fPIC -O2 -g -std=gnu99 $(VLC_CFLAGS) $(VLC_DEFINES)
+	$(CC) -c -o $@ $< $(CFLAGS) -fPIC -std=gnu99 $(VLC_CFLAGS) $(VLC_DEFINES)
 
 clean:
 	rm -f *.so
 	rm -f *.o
 
 install:
-	mkdir -p $(DESTDIR)$(PREFIX)/lib/vlc/plugins/audio_output
-	install -m755 $(TARGET_LIB) $(DESTDIR)$(PREFIX)/lib/vlc/plugins/audio_output
+	mkdir -p $(DESTDIR)$(PREFIX)/lib64/vlc/plugins/audio_output
+	install -m755 $(TARGET_LIB) $(DESTDIR)$(PREFIX)/lib64/vlc/plugins/audio_output
 
 uninstall:
 	rm -f $(DESTDIR)$(PREFIX)/lib/vlc/plugins/audio_output/$(TARGET_LIB)
diff -ru rsound-9999.old/src/Makefile rsound-9999/src/Makefile
--- rsound-9999.old/src/Makefile	2023-10-05 17:58:39.162759290 +0200
+++ rsound-9999/src/Makefile	2023-10-05 18:03:02.288739668 +0200
@@ -17,7 +17,7 @@
 TARGET_LIB_OBJ = librsound/librsound.o librsound/buffer.o
 TARGET_LIB_OBJ_STATIC = librsound/librsound.a
 
-PKGCONF_PATH = $(PREFIX)/lib/pkgconfig/rsound.pc
+PKGCONF_PATH = $(PREFIX)/lib64/pkgconfig/rsound.pc
 
 TARGET_SERVER_LIBS = -lpthread
 TARGET_LIB_LIBS = -lpthread
@@ -120,7 +120,7 @@
 $(TARGET_LIB): $(TARGET_LIB_OBJ) $(TARGET_LIB_OBJ_STATIC)
 	@echo "LD $@"
 ifeq ($(PLATFORM),OSX)
-	@$(CC) -dynamiclib -o $(TARGET_LIB) $(CFLAGS) $(TARGET_LIB_OBJ) $(TARGET_LIB_LIBS) -install_name $(PREFIX)/lib/librsound.dylib
+	@$(CC) -dynamiclib -o $(TARGET_LIB) $(CFLAGS) $(TARGET_LIB_OBJ) $(TARGET_LIB_LIBS) -install_name $(PREFIX)/lib64/librsound.dylib
 else
 	@$(CC) -shared -Wl,-soname,librsound.so.3 -o $(TARGET_LIB) $(CFLAGS) $(TARGET_LIB_OBJ) $(TARGET_LIB_LIBS) -fPIC
 	@ln -sf librsound.so.3.0.0 librsound/librsound.so.3
@@ -171,18 +171,18 @@
 
 
 install-lib: $(TARGET_LIB) $(TARGET_LIB_OBJ_STATIC) install-pkgconf
-	mkdir -p $(DESTDIR)$(PREFIX)/lib/pkgconfig 2>/dev/null || /bin/true
+	mkdir -p $(DESTDIR)$(PREFIX)/lib64/pkgconfig 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(PREFIX)/include 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1 2>/dev/null || /bin/true
-	mkdir -p $(DESTDIR)$(PREFIX)/lib/ckport/db 2>/dev/null || /bin/true
-	install -m755 $(TARGET_LIB) $(DESTDIR)$(PREFIX)/lib
-	install -m644 ../ckport/librsound.ckport $(DESTDIR)$(PREFIX)/lib/ckport/db
+	mkdir -p $(DESTDIR)$(PREFIX)/lib64/ckport/db 2>/dev/null || /bin/true
+	install -m755 $(TARGET_LIB) $(DESTDIR)$(PREFIX)/lib64
+	install -m644 ../ckport/librsound.ckport $(DESTDIR)$(PREFIX)/lib64/ckport/db
 ifneq ($(PLATFORM),OSX)
-	cp -P librsound/librsound.so librsound/librsound.so.3 $(DESTDIR)$(PREFIX)/lib
+	cp -P librsound/librsound.so librsound/librsound.so.3 $(DESTDIR)$(PREFIX)/lib64
 endif
-	install -m644 $(TARGET_LIB_OBJ_STATIC) $(DESTDIR)$(PREFIX)/lib
+	install -m644 $(TARGET_LIB_OBJ_STATIC) $(DESTDIR)$(PREFIX)/lib64
 	install -m644 librsound/rsound.h $(DESTDIR)$(PREFIX)/include
-	install -m644 rsound.pc $(DESTDIR)$(PREFIX)/lib/pkgconfig
+	install -m644 rsound.pc $(DESTDIR)$(PREFIX)/lib64/pkgconfig
 	install -m644 ../doc/rsd.1 $(DESTDIR)$(PREFIX)/share/man/man1
 	install -m644 ../doc/rsdplay.1 $(DESTDIR)$(PREFIX)/share/man/man1
 
@@ -197,14 +197,14 @@
 install-pkgconf:
 	@echo "prefix=$(PREFIX)" > rsound.pc
 	@echo "exec_prefix=\$${prefix}" >> rsound.pc
-	@echo "libdir=\$${prefix}/lib" >> rsound.pc
+	@echo "libdir=\$${prefix}/lib64" >> rsound.pc
 	@echo "includedir=\$${prefix}/include" >> rsound.pc
 	@echo "" >> rsound.pc
 	@echo "Name: RSound" >> rsound.pc
 	@echo "Description: An audio library designed for seamless networked transfer of audio streams." >> rsound.pc
 	@echo "Requires:" >> rsound.pc
 	@echo "Version: $(RSD_VERSION)" >> rsound.pc
-	@echo "Libs: -L\$${prefix}/lib -lrsound" >> rsound.pc
+	@echo "Libs: -L\$${prefix}/lib64 -lrsound" >> rsound.pc
 	@echo "Cflags: -I\$${prefix}/include" >> rsound.pc
 
 uninstall:
diff -ru rsound-9999.old/src/oss-emul/Makefile rsound-9999/src/oss-emul/Makefile
--- rsound-9999.old/src/oss-emul/Makefile	2023-10-05 17:58:39.163759290 +0200
+++ rsound-9999/src/oss-emul/Makefile	2023-10-05 18:04:00.552735323 +0200
@@ -3,8 +3,8 @@
 SCRIPT = ross
 
 all: librsoundoss.c
-	$(CC) $(CFLAGS) -g -O2 -Wall -lrsound -fPIC -shared -o $(TARGET) $(SOURCE)
+	$(CC) $(CFLAGS) -O2 -Wall -lrsound -fPIC -shared -o $(TARGET) $(SOURCE)
 
 install: all
-	install -m755 $(TARGET) /usr/lib
-	install -m755 $(SCRIPT) /usr/bin 
+	install -m755 $(TARGET) /usr/lib64
+	install -m755 $(SCRIPT) /usr/bin
